authorize_net.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |spec|
   spec.name = 'authorize_net'
-  spec.version = '0.1.2'
+  spec.version = '1.0.0'
   spec.date = '2016-06-04'
```

Comment:
@jvkoh should this date be updated?

Reply:
Looks like that probably shouldn't exist at all actually.

Date is determined automatically when you package a gem, and the only thing I can find about this variable says not to set it: https://ruby-doc.org/stdlib-2.3.0/libdoc/rubygems/rdoc/Gem/Specification.html

Reply:
Sounds like we should remove it then 😛